zenoh/src/value.rs

```diff
@@ -307,6 +595,38 @@ impl TryFrom<Value> for f64 {
     }
 }
 
+// f32 conversion
+impl From<f32> for Value {
+    fn from(f: f32) -> Self {
+        Value {
+            payload: ZBuf::from(f.to_string().as_bytes().to_vec()),
```

Comment:
Please construct the payload with ` ZBuf::from(Vec::<u8>::from(f.to_string()))` as in the other cases. I saw that also `f64` does it in this way. Do you mind updating it?

Reply:
Sure, I thought it was designed intentionally.
I'll update f32 and f64 just like the others.